src/command/client/search/history_list.rs

```diff
@@ -159,6 +159,11 @@ impl DrawState<'_> {
 
         for section in h.command.split_ascii_whitespace() {
             self.x += 1;
+            if (self.x > self.list_area.width) {
```

Comment:
```suggestion
            if self.x > self.list_area.width {
```
To make clippy happy :D